web/src/app/users/SendTestDialog.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import p from 'prop-types'
+
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/react-hooks'
+import Spinner from '../loading/components/Spinner'
+
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+} from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import DialogContentError from '../dialogs/components/DialogContentError'
+import toTitleCase from '../util/toTitleCase.ts'
+import { useConfigValue } from '../util/RequireConfig'
+
+const query = gql`
+  query($cmID: ID!, $number: String!) {
+    sendTestStatus(cmID: $cmID) {
+      details
+      status
+    }
+    userContactMethod(id: $cmID) {
+      type
+      formattedValue
+    }
+    phoneNumberInfo(number: $number) {
+      formatted
+    }
+  }
+`
+const useStyles = makeStyles({
+  statusOk: {
+    color: '#218626',
+  },
+  statusWarn: {
+    color: '#867321',
+  },
+  statusError: {
+    color: '#862421',
+  },
```

Comment:
Where are these from? is there a shared source we can use instead?

Reply:
these are the same colors from the alert log entries

Reply:
Maybe we can move them to be exported from `statusStyles.ts`?

Maybe something like


```ts
export const textColors = {
  statusOk: '#218626',
  // ...
}
```

and then just import for both places